airflow/bin/cli.py

```diff
@@ -130,6 +130,23 @@ class CLIFactory:
             choices=tabulate_formats,
             default="fancy_grid"),
 
+        # config
+        'section': Arg(
+            ("--section",),
+            help="Section of the config"),
+        'option': Arg(
+            ("--option",),
+            help="Option in the given section"),
+        'value': Arg(
+            ("--value",),
+            help="Value for the option"),
+        'executor': Arg(
+            ("--executor",),
+            help="Use to change the executor"),
+        'dagdir': Arg(
+            ("--dagdir",),
```

Comment:
```suggestion
            ("--dagdir",),
```
We are currently using the subdir option for this purpose.

Reply:
Yup, let's use `subdir`